compose/docker-compose.override.yml-all

```diff
@@ -90,7 +90,7 @@ services:
   # Varnish
   # -----------------------------------------------------------------------------------------------
   varnish:
-    image: devilbox/varnish:${VARNISH_SERVER:-6}
+    image: devilbox/varnish:${VARNISH_SERVER:-6}-0.3
```

Comment:
`$VARNISH_SERVER` is going to handle the full version numner: 4, 5 or 6. The sub-part I have included is for breaking changes.

**Without this subversion:**

Imagine I am changing and breaking the Varnish server (which pushes master and its current tags every night to Dockerhub), then the user would by (docker-compose pull) get the latest change, which will then break the setup.

**With the subversion:**

I can safely change and break the Varnish server (the current Devilbox attached version will stay unaffected, but still receive nightly updates by its old version). Once Varnish is changed, I will adjust it in the Devilbox repository, so it will work again and can PR a new change with an incremented sub-version. Once this is then merged, it works again. This will allow to go back and force on the Devilbox commits, with always working versions attached to it.

Reply:
OK thanks to clarify the version startegy !

Reply:
This is also similar to what is being done atm with Apache/Nginx and PHP:

* https://github.com/cytopia/devilbox/blob/master/docker-compose.yml#L98
* https://github.com/cytopia/devilbox/blob/master/docker-compose.yml#L213

Reply:
yes indeed :o)